src/PhpSpreadsheet/Shared/StringHelper.php

```diff
@@ -11,6 +11,11 @@ class StringHelper
     //    Fraction
     const STRING_REGEXP_FRACTION = '(-?)(\d+)\s+(\d+\/\d+)';
 
+    /**
+     * Maximum limit of chars in single cell in excel.
+     */
+    private const CELL_CHARS_LIMIT = 32767;
```

Comment:
Is there a particular reason why you've made this constant private rather than public?